portal/views/patch_flask_user.py

```diff
@@ -16,3 +19,39 @@ def patch_make_safe_url(url):
     safe_url = urlunsplit(
         (None, None, parts.path, parts.query, parts.fragment))
     return safe_url
+
+
+def patch_forgot_password():
+    """Need to customize flash message shown in forgot_password
+
+    No hooks available to customize the message, so this function is
+    intended to be a drop in replacement with only the text of the
+    message altered, as per TN-1030
+
+    """
+    """Prompt for email and send reset password email."""
+    user_manager = current_app.user_manager
+
+    # Initialize form
+    form = user_manager.forgot_password_form(request.form)
+
+    # Process valid POST
+    if request.method == 'POST' and form.validate():
+        email = form.email.data
+        user, user_email = user_manager.find_user_by_email(email)
+
+        if user:
+            user_manager.send_reset_password_email(email)
+
+        # Prepare one-time system message
+        flash(_("If the email address '%(email)s' is in the system, a "
+                "reset password email will now have been sent to it. "
```

Comment:
I might simplify the language a little bit, unless it's already been finalized:
>If an account exists for '%(email)s', an email to help reset the password was just sent to it

Reply:
yes, it feels wordy - that was the text from the story TN-1030.